server/src/commandLine.mjs

```diff
@@ -95,8 +98,15 @@ if ( enabledTriggerPaths.length > 0 ) {
   logger.info(`Triggers will be read from these paths: ${enabledTriggerPaths.join(', ')}`);
 }
 
+// --- novalidate method overrides
+let validateMethodOverrides = true;
+if( !config.dotConfig.supportedSdks[0].validateFlag && parsed.novalidate ){
```

Comment:
The validateFlag should be at the top-level of the dotConfig object, not within a supported SDK. So remove ".supportedSdks[0]".

Reply:
Done